cypress/integration/app_admin/app_admin.js

```diff
@@ -0,0 +1,45 @@
+import { Given, When, Then, Before, After } from 'cypress-cucumber-preprocessor/steps';
+
+const USER_NAME = 'test@unit.no';
+const APP_ADMIN_ROLE = 'App-admin';
+
+Before(() => {
+  cy.deleteUser(USER_NAME).then(() => {
+    cy.addUser(USER_NAME, APP_ADMIN_ROLE).then((userId) => {
+      cy.wrap(userId).as('userId');
+      cy.visit('/');
+      cy.wait(5000);

```

Comment:
vi burde kanskje prøve så godt vi kan å unngå fixed waits. testene vi har i frontend nå kan godt ta et kvarter++ å kjøre, selv etter vi har kuttet ut all fixed waiting.
Noen grunn til at det brukes her?

Reply:
Det er for å sikre at endringene i frontend er gjort. (meny for pålogget bruker f.eks)

Reply:
Det virker ikke uten? Ser vi ikke har noen eksplisitt `wait()` lenger i frontend, da cypress egentlig var ganske god til å vente på elementer selv (default er 4sec, kan overstyres). Da ventes det _maks_ så lenge, heller enn at det blir bastant venting selv om det ikke trengs

https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#When-Elements-Are-Missing

Reply:
Skal sjekke det, hadde ihvertfall problemer med det tidligere. Kan være at det at vi bruker mock i frontend gjør at det går fort nok.

Reply:
ja, det er sant... i så fall kan vi heller ha timeout på get() da, så slipper vi å vente i best-case

Reply:
Virker nå uten `wait`